sdk/src/main/java/com/meniga/sdk/models/accounts/enums/AccountCategory.java

```diff
@@ -9,21 +9,13 @@
  * Copyright 2017 Meniga Iceland Inc.
  */
 public enum AccountCategory implements Serializable {
-	@SerializedName("Wallet")
```

Comment:
Does it serialize correctly after this change?

Reply:
There's a separate model for the web service which has those annotations (this way we don't have a bloated model and we reduce chances of web model influencing our SDK model too much).